lib/main.js

```diff
@@ -0,0 +1,5 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import './safari-samesite.js';
+export {safariSameSiteFix} from './safari-samesite.js';
```

Comment:
Is there a need to export this as a "public" API?  I think just loading the module is the API?  Maybe this symbol should be `_safariSameSiteFix` (here and in the js file) and a comment added that it's a private API for testing?